olexandr.vakulinskiy/PR2/practice_2.rb

```diff
@@ -0,0 +1,55 @@
+module Notification
+  def self.log
```

Comment:
When I call this method I've got undefined method `log' for Email:Class (NoMethodError) error

Reply:
Ye, that is because of invalid syntax
Please take a look on this example
[grisha.pletin request](https://github.com/MastersAcademy/ruby-course-2020/pull/103/files) | notification_module.rb
Especially on nested `ClassMethods` module and self.included method.